### ISSUE TITLE
Colors get all scrambled when I try to assign them in my bed file in circos.genomicLink

### ISSUE THREAD

**Issue by lekha18** [jokergoo/circlize#132]
Hi,

I am trying to color my links using circos.genomicLink. I assigned colors in my bed file and used col=bed2$color. It works, I do get different colors but not the colors I want. 
If I only add two colors, any color I add becomes black and red. I assigned half my rows with blue and half with grey, and I still got half with red and half with black. If i use black, red and green then black is fine, but red becomes green and green becomes red. 
Why do you think that is? And is there a way to fix it?
Thanks!

**Comment by jokergoo** [comment]
Hi, is `color` column in `bed2` a factor?

try `col = as.character(bed2$color)`

**Comment by lekha18** [comment]
Hi, thank you! Yes, I made a column called color in my bed2 file and used that. I tried col = as.character(bed2$color) but it makes the colors very transparent.

![image](https://user-images.githubusercontent.com/54689994/64133673-1f44db80-ce2c-11e9-8820-0a4916b69654.png)

Tried to adjust the transparency but it gives the following erroe:
Error in circos.link(region1[i, 1], point1, region2[i, 1], point2, rou1 = rou1[i],  : 
  unused argument (transparency = 0)
